main.go

```diff
@@ -70,8 +73,10 @@ func main() {
 		MetricsBindAddress:     metricsAddr,
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
+		LeaderElection:         false,
```

Comment:
enabling leader election with the default lease/renewal works for me. What error were you hitting?